invenio_app_ils/records/views.py

```diff
@@ -129,6 +128,17 @@ def create_document_request_action_blueprint(app):
     return blueprint
 
 
+class UserInfoResource(UserInfoView):
+    """Retrieve current user's information."""
+
+    def response(self, user):
+        """."""
+        user_payload = default_user_payload(user)
+        user_payload["roles"] = [role.name for role in user.roles]
+        user_payload.update(dict(locationPid='1', username=user.email))
```

Comment:
I think it would probably better to define a `ILS_DEFAULT_LOCATION_PID` than hardcoding `1`, especially because it has a base32 PID, not it, no?